content/en/blog/_posts/2020-08-31-increase-kubernetes-support-one-year.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blog 
+title: 'Increase Kubernetes Support Window to One Year'
+date: 2020-08-31 
+slug: kubernetes-1-19-feature-one-year-support
+---
+
+**Authors:** Tim Pepper (VMware), Nick Young (VMware)
+
+Starting with Kubernetes 1.19, the support window for Kubernetes versions [will increase from three versions](https://github.com/kubernetes/enhancements/issues/1498) (usually nine months support) to one year. The longer support window is intended to allow organisations to perform major upgrades at whatever time of the year works the best for them.
+
+This is a big change. For many years the Kubernetes project has delivered a new minor release (e.g.: 1.13 or 1.14) every 3 months. The project provides bugfix support via patch releases (e.g.: 1.13.Y) for three parallel branches of the codebase. Combined, this led to each minor release (e.g.: 1.13) having a patch release stream of support for approximately 9 months. In the end a cluster operator must upgrade at least every 9 months to remain supported. But nine months is an irregular period of time relative to the planning, operations, and financial cycles of most companies.
+
+A survey conducted in early 2019 by the WG LTS showed that a significant subset of Kubernetes end-users fail to upgrade within the 9-month support period. 
+
+![Versions in Production](/images/blog/2020-08-31-increase-kubernetes-support-one-year/versions-in-production-text-2.png)
+
+This, and other responses from the survey, suggest that a considerable portion of our community would better be able to lifecycle manage their deployments on supported versions if the patch support period were extended to 12-14 months. It appears to be true regardless of whether the users are on DIY build or commercially vendored distributions. An extension in the patch support length of time would thus lead to a larger percentage of our user base running supported versions compared to what we have now.
+
+A yearly support period provides the cushion end-users appear to desire, and is more in harmony with familiar annual planning cycles.
+We view elongating the patch support window toward 1 year as a relatively minor change, yet is believed viable from an implementation perspective and is a meaningful first response to users.
```

Comment:
```suggestion
```
I would remove this sentence, it doesn't flow well and seems a little redundant with the previous sentence.